bin/GoogleAdsCleanupServices.php

```diff
@@ -0,0 +1,529 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Util;
+
+use Composer\Script\Event;
+
+/**
+ * Utilities to remove Google Ads API services in the library.
+ */
+class GoogleAdsCleanupServices {
+
+	/**
+	 * Display debug output.
+	 *
+	 * @var boolean
+	 */
+	protected $debug = false;
+
+	/** List of services to keep.
+	 *
+	 * @var array
+	 */
+	protected $services_keep = [
+		'AccountBudget',
+		'AccountLink',
+		'Ad',
+		'AdGroup',
+		'AdGroupAdLabel',
+		'AdGroupAd',
+		'AdGroupCriterion',
+		'AssetGroup',
+		'AssetGroupListingGroupFilter',
+		'BillingSetup',
+		'Campaign',
+		'CampaignBudget',
+		'CampaignCriterion',
+		'ConversionAction',
+		'Customer',
+		'CustomerUserAccess',
+		'GeoTargetConstant',
+		'GoogleAds',
+		'MerchantCenterLink',
+	];
+
+	/**
+	 * List of services to remove.
+	 *
+	 * @var array
+	 */
+	protected $services = [
+		'AccountBudgetProposal',
+		'AdGroupAdAssetView',
+		'AdGroupAudienceView',
+		'AdGroupBidModifier',
+		'AdGroupCriterionCustomizer',
+		'AdGroupCriterionLabel',
+		'AdGroupCriterionSimulation',
+		'AdGroupCustomizer',
+		'AdGroupExtensionSetting',
+		'AdGroupFeed',
+		'AdGroupLabel',
+		'AdGroupSimulation',
+		'AdParameter',
+		'AdScheduleView',
+		'AgeRangeView',
+		'Asset',
+		'AssetFieldTypeView',
+		'AssetGroupProductGroupView',
+		'AssetSet',
+		'BatchJob',
+		'BiddingDataExclusion',
+		'BiddingSeasonalityAdjustment',
+		'BiddingStrategy',
+		'BiddingStrategySimulation',
+		'CallView',
+		'CampaignAsset',
+		'CampaignAssetSet',
```

Comment:
That's a good point. The code for removing services was written a while back when we didn't have full testing in place, so I didn't want a missed import to cause the service to be removed from the library. Now that things have matured it might be better to implement a similar method as for the Enums. It will definitely make it easier to maintain, in case we update the library to a newer version.